server/templates/server/help_table.html

```diff
@@ -255,6 +255,10 @@ <h4 class="modal-title w-100 font-weight-bold">בחר את הפילטרים הר
                                     </div>
 
                                 </div>
+                                  <div>
+                                      <p class="h5"><u>בחר עיר</u></p>
```

Comment:
the <u> tag is deprecated, use "text-decoration: underline" attribute in CSS instead